# HG changeset patch
# Parent b3a1518094c954c1ad876bbf7fe64893ad725232
# User Frédéric Wang <fred.wang@free.fr>
MathML Operator Dictionary: fix three conflicts for symmetric or fence (bug 534970). r=karlt

diff --git a/layout/mathml/mathfont.properties b/layout/mathml/mathfont.properties
--- a/layout/mathml/mathfont.properties
+++ b/layout/mathml/mathfont.properties
@@ -163,17 +163,17 @@ operator.\u005B.prefix = lspace:0 rspace
 operator.\u005C.infix = lspace:0 rspace:0 # reverse solidus
 operator.\u005D.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # ]
 operator.\u005E.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &Hat; circumflex accent
 operator.\u005E.infix = lspace:1 rspace:1 direction:horizontal # ^
 operator.\u005F.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # _ low line
 operator.\u005F.infix = lspace:2 rspace:2 stretchy direction:horizontal # _
 operator.\u0060.postfix = lspace:0 rspace:0 accent # &DiacriticalGrave;
 operator.\u007B.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # {
-operator.\u007C.infix = lspace:0 rspace:0 stretchy symmetric direction:vertical # &VerticalLine; |
+operator.\u007C.infix = lspace:2 rspace:2 stretchy fence symmetric direction:vertical # &VerticalLine; |
 operator.\u007C.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # |
 operator.\u007C.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # |
 operator.\u007C\u007C.infix = lspace:4 rspace:4 direction:vertical # ||
 operator.\u007C\u007C.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # multiple character operator: ||
 operator.\u007C\u007C.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # multiple character operator: ||
 operator.\u007C\u007C\u007C.infix = lspace:2 rspace:2 stretchy fence symmetric direction:vertical # multiple character operator: |||
 operator.\u007C\u007C\u007C.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # multiple character operator: |||
 operator.\u007C\u007C\u007C.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # multiple character operator: |||
@@ -200,18 +200,18 @@ operator.\u02D8.postfix = lspace:0 rspac
 operator.\u02D9.postfix = lspace:0 rspace:0 accent # &DiacriticalDot;
 operator.\u02DA.postfix = lspace:0 rspace:0 accent # ring above
 operator.\u02DC.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &DiacriticalTilde; small tilde
 operator.\u02DD.postfix = lspace:0 rspace:0 accent # &DiacriticalDoubleAcute;
 operator.\u02F7.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # modifier letter low tilde
 operator.\u0302.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # combining circumflex accent
 operator.\u0311.postfix = lspace:0 rspace:0 accent # &DownBreve;
 operator.\u03F6.infix = lspace:5 rspace:5 # greek reversed lunate epsilon symbol
-operator.\u2016.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &Vert; &Verbar;
-operator.\u2016.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &Vert; &Verbar;
+operator.\u2016.prefix = lspace:0 rspace:0 stretchy fence direction:vertical # &Vert; &Verbar;
+operator.\u2016.postfix = lspace:0 rspace:0 stretchy fence direction:vertical # &Vert; &Verbar;
 operator.\u2018.prefix = lspace:0 rspace:0 fence # &OpenCurlyQuote;
 operator.\u2019.postfix = lspace:0 rspace:0 fence # &CloseCurlyQuote;
 operator.\u201C.prefix = lspace:0 rspace:0 fence # &OpenCurlyDoubleQuote;
 operator.\u201D.postfix = lspace:0 rspace:0 fence # &CloseCurlyDoubleQuote;
 operator.\u2022.infix = lspace:4 rspace:4 # bullet
 operator.\u2026.infix = lspace:0 rspace:0 # horizontal ellipsis
 operator.\u2032.postfix = lspace:0 rspace:2 # prime
 operator.\u203E.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # overline
